github_hooks/db/migrate/20250409112220_add_deleted_at_for_organizations.rb

```diff
@@ -0,0 +1,5 @@
+class AddDeletedAtForOrganizations < ActiveRecord::Migration[5.1]
+  def change
+    add_column :organizations, :deleted_at, :datetime, null: true, default: nil
```

Comment:
in projects we added "deleted_by". That seems like a good idea, why not add it here as well?